card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -57,6 +57,25 @@ def wrap_item item, args={}
     #.merge(:pointer_item_class=>'form-control')))
   end
 
+  view :single_autocomplete do |args|
+    args ||= {}
+    items = args[:item_list] || card.item_names(:context=>:raw)
+    items = [''] if items.empty?
+    options_card_name = (oc = card.options_rule_card) ? oc.cardname.url_key : ':all'
+
+    extra_css_class = args[:extra_css_class] || 'pointer-list-ul'
+
+    %{
+      <ul class="pointer-list-editor #{extra_css_class}" data-options-card="#{options_card_name}">
+        <li class="pointer-li">
+        <span class="input-group">
+          #{ text_field_tag 'pointer_item', items[0], :class=>'pointer-item-text form-control' }
+        </span>
+        </li>
+      </ul>
+    }
+  end
+
```

Comment:
I see two possibilities to reduce code repetition:
1) move all the logic of line 61-66 to `default_list_args` and store it in args and then call `default_list_args` in `default_single_autocomplete_args`
or 2) make the drag&drop handle, the remove and the add button optional in `list_item` and `list` view and turn it off in in  `default_single_autocomplete_args`. Then you just call `list` view in `single_autocomplete` view